### PR TITLE
GRW-2011Add cart page desktop layout

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Add styles for cart page desktop layout. The styles are the same as the checkout page since these two pages have a lot in common. 

Screenshot 2023-01-03 at 14 55 59

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-2011]: https://hedvig.atlassian.net/browse/GRW-2011?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 1]
```diff
@@ -102,48 +102,78 @@ const EmptyState = ({ children, prevURL }: EmptyStateProps) => {
...
           </Link>
         </Space>
-
         {children}
       </Space>
     </Wrapper>
   )
 }
 
+const CenteredText = styled(Text)({ textAlign: 'center' })
```
Comment (alebedev) [inline apps/store/src/components/CartPage/CartPage.tsx:111]: Maybe we should support this in Text props?

**Review by alebedev — APPROVED** [review]
Looks good

**Review by amandaflod — APPROVED** [review]
Nice!

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 2]
```diff
@@ -102,48 +102,78 @@ const EmptyState = ({ children, prevURL }: EmptyStateProps) => {
...
+
+const HorizontalLine = styled.hr(({ theme }) => ({
+  backgroundColor: theme.colors.gray300,
+  height: 1,
+}))
+
+const Wrapper = styled(Space)(({ theme }) => ({
+  paddingBottom: theme.space[6],
```
Comment (robinandeer) [inline apps/store/src/components/CartPage/CartPage.tsx:119]: nit: `theme.space.xl`

**Review by robinandeer — APPROVED** [review]
Looks great!

[clappingwtf](https://app.graphite.dev/macros/HedvigInsurance)

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply apps/store/src/components/CartPage/CartPage.tsx:111]: good catch, it's already supposed I just overlooked this one.

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 2 above)
Comment (KarnellSchultz) [reply apps/store/src/components/CartPage/CartPage.tsx]: tack tack - swapped those tokens.

**Pull request merged**